library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -3052,7 +3052,7 @@ def load_config
     def autoeject
       load_config unless @config
 
-      @config[autoeject] || false
+      @config["automatic_eject"] || false
```

Comment:
Better `@config.fetch("automatic_eject", false)` otherwise people may change the default value to `|| true` and get a surprise.